src/native/libs/System.Net.Security.Native/pal_gssapi.c

```diff
@@ -56,6 +56,13 @@ static gss_OID_desc gss_mech_ntlm_OID_desc = {.length = STRING_LENGTH(gss_ntlm_o
                                               .elements = gss_ntlm_oid_value};
 #endif
 
+#if !HAVE_GSS_C_INQ_SSPI_SESSION_KEY
```

Comment:
I'm not 100% sure what I'm doing here. It seems like MIT krb5, Heimdal, and GSS Framework do define `GSS_C_INQ_SSPI_SESSION_KEY` and works when I compile it from the `src/native/libs` dir but when I tried to compile .NET from the root project I still needed to manually define it. In the end it didn't seem an issue to just redefine if needed but happy to change this around if needed.